OpenRA.Mods.Common/Traits/Render/RenderDetectionCircle.cs

```diff
@@ -19,14 +19,35 @@ namespace OpenRA.Mods.Common.Traits
 {
 	class RenderDetectionCircleInfo : ITraitInfo, Requires<DetectCloakedInfo>
 	{
-		public object Create(ActorInitializer init) { return new RenderDetectionCircle(init.Self); }
+		[Desc("Draw a rotating radar scanner update line, disabled by default.")]
+		public readonly bool DrawUpdateLine = false;
+
+		[Desc("Number of (angle / 1024) the Radar update line advances per tick.")]
```

Comment:
It's a negative angle - which results with a counterclockwise animation. It's not a magic value.